### PR TITLE
Config command issue fixed

### PR DESCRIPTION
**Pull request by vimalraghwani20**
# Problem Statement
Config command issue fixed

# Coding 
Fixed the index name of config file 

# Testing 
[OpenldapProvisioning.log](https://github.com/Seagate/cortx-utils/files/6876379/OpenldapProvisioning.log)

### CODE REVIEW

File: py-utils/src/utils/setup/openldap/setupcmd.py [hunk 1]
```diff
@@ -54,9 +54,9 @@ class SetupCmd(object):
   Log.init('OpenldapProvisioning','/var/log/seagate/utils/openldap',level='DEBUG')
 
   config_file_path = "/etc/cortx/cortx.conf"
-  Conf.load('config_file', f'yaml:///{config_file_path}')
-  openldap_prov_config = Conf.get(index='config_file', key='install_path') + '/cortx/utils/conf/' + 'openldap_prov_config.yaml'
-  openldap_config_file = Conf.get(index='config_file', key='install_path') + '/cortx/utils/conf/' + "openldap_config.yaml"
+  Conf.load('config_file_install_path', f'yaml:///{config_file_path}')
```
Comment (shalakadharap) [inline py-utils/src/utils/setup/openldap/setupcmd.py:57]: No its not required. Its already loaded by you in setupcmd.py so
Just remove lines 57 and 58 and keep just below-
mdb_dir = Conf.get(index='config_file', key='install_path') + '/cortx/utils/conf'

Comment (vimalraghwani20) [reply py-utils/src/utils/setup/openldap/setupcmd.py]: I can not remove the load config from the setup cmd as it is base class for all the phases. 
I have remove the load config from the base_configure_ldap.py

File: py-utils/src/utils/setup/openldap/base_configure_ldap.py [hunk 2]
```diff
@@ -92,9 +92,7 @@ def performbaseconfig(rootpassword, forcecleanup, config_values):
             quit()
         if forcecleanup != None :
             forceclean = forcecleanup
-        config_file_path = "/etc/cortx/cortx.conf"
-        Conf.load('config_file', f'yaml:///{config_file_path}')
-        mdb_dir = Conf.get(index='config_file', key='install_path') + '/cortx/utils/conf'
+        mdb_dir = Conf.get(index= 'util_config_file_index', key='install_path') + '/cortx/utils/conf'
```
Comment (shalakadharap) [inline py-utils/src/utils/setup/openldap/base_configure_ldap.py:95]: keep index name as it is. Line will be -
mdb_dir = Conf.get(index='config_file', key='install_path') + '/cortx/utils/conf'

**Review by shalakadharap — APPROVED** [review]

**Review by shahidSTX — APPROVED** [review]
LGTM

**Pull request merged**